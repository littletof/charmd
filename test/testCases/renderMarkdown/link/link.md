[Deno](https://deno.land)
[Bottom](#bottom)
[LinkReference]

[LinkReference]: https://deno.land/x

[link with title](https://deno.land "title text!")
Autoconverted link https://deno.land