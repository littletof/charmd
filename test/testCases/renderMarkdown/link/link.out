[36m[Deno](https://deno.land)[39m
[36m[Bottom](#bottom)[39m
[36m[3m[LinkReference][23m[39m

[90m[3m[36m[LinkReference][90m: [](https://deno.land/x)
[23m[39m
[36m[link with title](https://deno.land)[39m
Autoconverted link https://deno.land
